kakarot_scripts/utils/kakarot.py

```diff
@@ -624,10 +637,15 @@ async def eth_get_code(address: Union[int, str]):
     )
 
 
-async def deploy_presigned_tx(deployer: int, signed_tx: bytes, amount=0.1, name=""):
-    deployer_starknet_address = await deploy_and_fund_evm_address(deployer, amount)
+async def deploy_presigned_tx(
```

Comment:
what does "deploying" a tx mean?

Reply:
deploy a contract using a preseigned tx. needs to 
1. fund the deployer address
2. send the presigned tx
3. return the deployed contract addresses

Reply:
` deploy_with_presigned_tx` maybe?